Gems/AtomLyIntegration/EMotionFXAtom/Code/Source/AtomActorDebugDraw.cpp

```diff
@@ -427,6 +472,10 @@ namespace AZ::Render
         EMotionFX::JointSelectionRequestBus::BroadcastResult(
             cachedSelectedJointIndices, &EMotionFX::JointSelectionRequests::FindSelectedJointIndices, instance);
 
+        size_t cachedHoveredJointIndex;
```

Comment:
update: `cachedHoveredJointIndex` should be initialized to `0` (or some invalid value we can check against if the bus call fails for some reason)

Reply:
Fixed